src/main/scala/io/ino/sbtpillar/Plugin.scala

```diff
@@ -89,7 +89,7 @@ object Plugin extends sbt.Plugin {
 
   def pillarSettings: Seq[sbt.Def.Setting[_]] = inConfig(Test)(taskSettings) ++ taskSettings
 
-  private case class CassandraUrl(hosts: Seq[String], port: Int, keyspace: String)
+  private case class CassandraUrl(hosts: Seq[String], port: Int, keyspace: String, var username: String = "cassandra", var password: String = "cassandra")
```

Comment:
What about an `Option[Credentials]` instead, which could be used to do s.th. like `url.credentials.foreach(cred => cluster.withCredentials(cred.username, cred.password)` when building the `Cluster` below?